app/controllers/application_controller.rb

```diff
@@ -54,6 +54,7 @@ def google_register
 
   def logout
     log_out
+    p 'test'
```

Comment:
I guess we don't need that `test` here ;)

Reply:
That's true! I did not pay attention! Thanks for pointing that out!